ui/styl/fonts.styl

```diff
@@ -0,0 +1,22 @@
+// partially copied from https://github.com/code42day/stylus-font-face
+// TODO: restrict glyph ranges and inline fonts
+// TODO: possibly include other font types (eg ttf)
+file(name, ext, path)
+  path + '/' + name + '.' + ext
+
+font-face(family, style='normal', font=family, path='fonts', weight='normal')
```

Comment:
why is 'font' distinct from 'family'?  Should this perhaps be filename or something?  we're not even using it, it can probably be removed.